ehitchhiking-ui/src/app/shared/components/rating/rating.component.sass

```diff
@@ -0,0 +1,34 @@
+fieldset
+  border: none
+
+.rating
+  float: left
+
+.rating:not(:checked) > input
+  position: absolute
+  clip: rect(0,0,0,0)
+
+.rating:not(:checked) > label
+  float: right
+  width: 1em
+  padding: 0.1em
+  overflow: hidden
+  white-space: nowrap
+  cursor pointer
+  font-size: 100%
+  line-height: 1.2
```

Comment:
Invalid value. Unit of measurement isn't specified

Reply:
fixed